resources/icons/cancel.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
this is already in develop :main-icons/cancel

Reply:
that's part of another PR, fixed here https://github.com/status-im/status-react/pull/7396/files